### ISSUE TITLE
sequence labeling for sentences and not tokens

### ISSUE THREAD

**Issue by antgr** [huggingface/transformers#2724]
# ❓ Questions & Help

## Details

I have sentences that belong to a paragraph. Each sentence has a label.
[s1,s2,s3,..], [l1,l2,l3,...]
I understand that I have to encode each sentence using an encoder e.g bert, and then use sequence labeling. Could you guide me on how I could do that, combining them?

**A link to original question on Stack Overflow**: 
https://stackoverflow.com/questions/60048900/sequence-labeling-for-sentences-and-not-tokens

**Comment by LysandreJik** [comment]
Hi! You could leverage one of the `XXXForSequenceClassification` models for this. Their purpose is to classify sequences into a given number of labels. You would need to initialize a model from a pre-trained checkpoint:

```py
from transformers import BertForSequenceClassification

model = BertForSequenceClassification.from_pretrained("bert-base-cased")
```

This instantiates the base transformer model, but doesn't instantiate the classifier layer on top, you would need to train that with a fine-tuning on your own specific task.

**Comment by antgr** [comment]
Does the fact that I want to clarify entire sentences and not words, makes any difference? And if yes what is this difference? Is there any example with this spesific use case?
